Lib/test/test_xml_etree.py

```diff
@@ -2466,9 +2466,9 @@ def test_findall_different_nsmaps(self):
         nsmap = {'xx': 'X', None: 'Y'}
         self.assertEqual(len(root.findall(".//xx:b", namespaces=nsmap)), 2)
         self.assertEqual(len(root.findall(".//b", namespaces=nsmap)), 1)
-        nsmap = {'xx': 'X', '': 'Y'}
-        with self.assertRaisesRegex(ValueError, 'namespace prefix'):
-            root.findall(".//xx:b", namespaces=nsmap)
+        #nsmap = {'xx': 'X', '': 'Y'}
+        #with self.assertRaisesRegex(ValueError, 'namespace prefix'):
```

Comment:
Commented test sometimes causes confusion over it's existence. Perhaps remove the test since tbe `ValueError` is removed?

Reply:
I was planning to bring back the functionality ASAP, probably as a (deprecation?) warning. But I guess it's better to have a clean removal for now, as the discussion what to do about it is not urgent.